payjoin/src/send/error.rs

```diff
@@ -269,19 +269,24 @@ impl From<InternalCreateRequestError> for CreateRequestError {
     fn from(value: InternalCreateRequestError) -> Self { CreateRequestError(value) }
 }
 
-/// Represent an error returned by the receiver.
+/// Represent an error returned by Payjoin receiver.
 pub enum ResponseError {
-    /// `WellKnown` errors following the BIP78 spec
-    /// https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki#user-content-Receivers_well_known_errors
-    /// These errors are displayed to end users.
+    /// `WellKnown` Errors are defined in the [`BIP78::ReceiverWellKnownError`] spec.
```

Comment:
I find the use of `::` to identify the receiver spec a bit odd since it's not referencing a rust module element but it does get the point across